fe/fe-core/src/main/java/com/starrocks/sql/StatementPlanner.java

```diff
@@ -68,7 +50,7 @@ public ExecPlan plan(StatementBase stmt, ConnectContext session) throws Analysis
             Map<String, Database> dbs = AnalyzerUtils.collectAllDatabase(session, insertStmt);
             try {
                 lock(dbs);
-                return createInsertPlan(relation, session);
+                return new InsertPlanner().plan((InsertStmt) stmt, session);
```

Comment:
InsertStmt do not have InsertRelation, but QueryStatement have QueryRelation?

Reply:
Yes, InsertRelation is a wrong design, and InsertStatement is used here to unify. There is only relation in query, other statements do not contain relation